{
  width2: 160, height2: 120,
  D2: [0.0, 0.0, 0.0, 0.0, 0.0],
  K2: [142.58555603, 0.0, 78.5, 0.0, 142.58555603, 58.75, 0.0, 0.0, 1.0],
  R2: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0],
  P2: [142.58555603, 0.0, 78.5, 0.0, 0.0, 142.58555603, 58.75, 0.0, 0.0, 0.0, 1.0, 0.0]
}
